fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -220,7 +220,13 @@ def apply_interface_data(self, interface_node, port):
         port.description = resolve_new_value(interface_node, "description", port.description)
 
         if first(interface_node.xpath("disable")) is not None:
-            port.shutdown = True if resolve_operation(first(interface_node.xpath("disable"))) != "delete" else False
+            if resolve_operation(first(interface_node.xpath("disable"))) != "delete":
+                port.shutdown = True
+            else:
+                if port.shutdown is True:
+                    port.shutdown = False
+                else:
+                    raise NotFound('')
```

Comment:
It would be cool, however a real call to the switch replies (below), so we have to be on par with his return.
```
<rpc-reply xmlns="urn:ietf:params:xml:ns:
netconf:base:1.0" xmlns:junos="http://xml.juniper.net/junos/14.1X53/junos" xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:3574f938-ebbf-11e5-bd34-4aa91c02a53f">
<rpc-error>
<error-severity>warning</error-severity>
<error-path>[edit interfaces xe-0/0/9]</error-path>
<error-message>statement not found: </error-message>
</rpc-error>
<load-error-count>1</load-error-count>
</rpc-reply>
```